cloud_governance/common/clouds/aws/resource_explorer/resource_explorer_operations.py

```diff
@@ -0,0 +1,52 @@
+import boto3
+
+from cloud_governance.common.logger.init_logger import logger
+
+# @Todo, This class will be used in the feature releases.
+# @Todo, it helps in find the resource data like tags by using search query
+
+
+class ResourceExplorerOperations:
+    """
+    This class performs the resource explorer operations
+    """
+
+    def __init__(self):
+        self.__client = self.__set_client()
+
+    def __set_client(self):
+        view = self.list_views()
+        region = 'us-east-1'
```

Comment:
I think should be 1 variable for all places
Do we have default region in env variable ?